lib/rpc/ProviderHealthStateDynamoDbRepository.ts

```diff
@@ -0,0 +1,44 @@
+import { ProviderHealthStateRepository } from './ProviderHealthStateRepository'
+import Logger from 'bunyan'
+import { DynamoDB } from 'aws-sdk'
+import { DocumentClient } from 'aws-sdk/clients/dynamodb'
+import { ProviderHealthState } from './ProviderHealthState'
+
+export class ProviderHealthStateDynamoDbRepository implements ProviderHealthStateRepository {
```

Comment:
no unit test ?

Reply:
There is `test/mocha/integ/rpc/ProviderHealthStateDynamoDbRepository.test.ts` down below.

Reply:
that's integration test

Reply:
Although the folder name is `integ`, they are actually unit tests, just with the usage of DynamoDB.